docs/error-reporting/platform-integrations/source-map.md

```diff
@@ -139,14 +186,14 @@ module.exports = {
 }
 ```
 
-If you're using code transpiler plugins (such as Typescript), ensure to enable source-mapping there as well.
+If you're using code transpiler plugins (such as Typescript), be sure to enable source maps there as well.
 
 **Step 2: Set up `@backtrace-labs/webpack-plugin`**
 
 1. Install `@backtrace-labs/webpack-plugin` as a developer dependency:
 
    ```bash
-   npm install --save-dev @backtrace-labs/webpack-plugin
+   npm install -g --save-dev @backtrace-labs/webpack-plugin
```

Comment:
we don't need to use -g option with transpilers like webpack - you don't need to install it globally. It would be even better to install it locally instead.